c++/src/capnp/compiler/generics.c++

```diff
@@ -564,7 +564,15 @@ kj::Maybe<BrandedDecl> BrandScope::compileDeclExpression(
       }
 
       KJ_IF_MAYBE(r, resolver.resolve(nameValue)) {
-        return interpretResolve(resolver, *r, source);
+        auto result = interpretResolve(resolver, *r, source);
+        if (r->is<Resolver::ResolvedDecl>()){
+          errorReporter.reportResolution(Resolution {
+            .startByte = source.getStartByte(),
+            .endByte = source.getEndByte(),
+            .target = Resolution::Type { r->get<Resolver::ResolvedDecl>().id },
+          });
```

Comment:
Ugh, I forgot that the master branch is still on C++17 and it doesn't support designated initializers (although only MSVC actually complains)...

```suggestion
          errorReporter.reportResolution(Resolution {
            /* .startByte = */ source.getStartByte(),
            /* .endByte = */ source.getEndByte(),
            /* .target = */ Resolution::Type { r->get<Resolver::ResolvedDecl>().id },
          });
```